package.json

```diff
@@ -8,21 +8,21 @@
     "build": "yarn workspace @hashicorp/consul-ui-toolkit run build",
     "changeset": "changeset",
     "cleanup:local:toolkit": "yarn workspace @hashicorp/consul-ui-toolkit run cleanup:local",
-    "lint": "yarn workspaces run lint",
-    "lint:fix": "yarn workspaces run lint:fix",
+    "lint": "yarn workspaces foreach -A run lint",
+    "lint:fix": "yarn workspaces foreach -A run lint:fix",
     "prepare": "yarn build",
     "prerelease": "yarn prepare",
     "publish:local:toolkit": "yarn workspace @hashicorp/consul-ui-toolkit run publish:local",
     "release": "changeset publish",
     "start": "concurrently 'npm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
     "start:addon": "yarn workspace @hashicorp/consul-ui-toolkit run start",
     "start:documentation": "yarn workspace documentation run start",
-    "test": "yarn workspaces run test",
+    "test": "yarn workspaces -A run test",
```

Comment:
My googling was insufficient, when would I reach for `yarn workspaces -A` vs `yarn workspaces foreach -A`?

Reply:
Sorry - from a commit view. I see this isn't relevant in full diff